src/stub/core_impl.rs

```diff
@@ -250,6 +250,8 @@ impl<T: Target, C: Connection> GdbStubImpl<T, C> {
 // detail. The alternative would be to use multiple `bool` fields, which wastes
 // space in minimal `gdbstub` configurations.
 bitflags::bitflags! {
+    #[derive(Copy, Clone, Debug, Eq, Hash, Ord, PartialEq, PartialOrd)]
```

Comment:
Given that this is an internal type, I think you should be safe in removing all these extra derives

Reply:
Done.